main/tracking/courseLog.php

```diff
@@ -605,6 +605,7 @@ function(index) {
             20,
             'users_tracking'
         );
+        $table->direction = $trackingDirection ? $trackingDirection : 'ASC';
```

Comment:
This fix is onyl for the current table, but in the constructor (for parent constructor) from SortableTableFromArray class the argument for the direction is not set.

It would be nice to correct it from the same class for all the tables that are used

Reply:
It's done, thanks for the recommendation